spec/scrabble_spec.rb

```diff
@@ -0,0 +1,72 @@
+require './lib/scrabble.rb'
+
+describe Scrabble do
+
+  context "Scoring words" do
+
```

Comment:
I'm still working out how to organize the tests.  I was organizing them by method but putting the errors in their own context maybe means I need to look at possibly taking that part out of my self.score method and making a new one.